src/services/swap_service.ts

```diff
@@ -99,7 +99,12 @@ export class SwapService {
             makerAssetAmount,
             totalTakerAssetAmount,
             protocolFeeInWeiAmount: protocolFee,
+            gas,
         } = attributedSwapQuote.bestCaseQuoteInfo;
+        const {
+            makerAssetAmount: minMakerAssetAmount,
+            totalTakerAssetAmount: minTotalTakerAssetAmount,
```

Comment:
If this was a buy, wouldn't this be a `maxTotalTakerAssetAmount`? Maybe it makes more sense to just prefix these with `worst` instead of `min`.